src/routes/session.ts

```diff
@@ -46,6 +46,14 @@ export const login: RequestHandler = (req, res) => {
 						resource: 'user',
 						id: user.id,
 						passthrough: { req: root },
+						options: {
+							$filter: {
+								$or: {
+									password_reset_code: { $ne: null },
+									can_reset_password_until__expiry_date: { $ne: null },
```

Comment:
These fields don't exist on the open source model, see the comment below. I don't think filtering on them will work.